src/main/java/org/usfirst/frc4904/robot/commands/FlywheelSpinDown.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Flywheel;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FlywheelSpinDown extends CommandBase {
+  public static final double DEFAULT_OFF_SPEED = 0.0;
+  protected final Flywheel flywheel;
+
+  /**
+   * Spin down the flywheel
+   * 
+   * @param flywheel The flywheel to manipulate
+   */
+  FlywheelSpinDown(Flywheel flywheel) {
+    super();
+    setName("FlywheelSpinDown");
+    addRequirements(flywheel);
+    this.flywheel = flywheel;
+  }
+
+  public void execute() {
+    this.flywheel.setSpeed(DEFAULT_OFF_SPEED);
```

Comment:
don't need `this.`, make sure to `@Override`
Also shouldn't this be in `initialize`?

Reply:
Nikhil said it should be in `execute` because we need `Motor.set()` every "frame". I will make the other changes.